doc/publishing-open-data-in-fiware/how-to-publish-open-datasets-in-ckan-2.md

```diff
@@ -23,23 +23,49 @@ the visibility of the dataset:
     only some selected users will be able to access the dataset.
 -   Searchable: you can choose if you want your dataset to be displayed
     in the queries performed by the FIWARE Data portal users.
-    -   ​This field is only enabled when “Visibility” is set to
+    -   This field is only enabled when “Visibility” is set to
         “Private”.
     -   If you create a public dataset, it will be always searchable.
 -   Allowed Users: the list of users that can access your dataset.
     -   This field is only enabled when “Visibility” is set to
-        “Private”.​
+        “Private”.
 
-​​[![HowToPublishDatasheetsInCkan2](images/HowToPublishDatasheetsInCkan21.png)](images/HowToPublishDatasheetsInCkan21.png)
+[![HowToPublishDatasheetsInCkan2](images/HowToPublishDatasheetsInCkan21.png)](images/HowToPublishDatasheetsInCkan21.png)
 
 In the second step you have to provide the data itself. To do that, you can
 provide a link or upload a file with the data. Any type of file is allowed, but if you
 want to generate an automatic API to access your data, you must upload a
 CSV file.
 
+> **NOTE**: Take into account that “Private” resources created using the link
+> option must be secured using the appropiated mechanism (e.g. by following the
+> [Handling authorization and access control to APIs][] tutorial). CKAN will
+> only control the access using the CKAN interface, but direct access to the
+> linked resource must be controlled on the resource itself.
+
+
 [![HowToPublishDatasheetsInCkan3](images/HowToPublishDatasheetsInCkan3.png)](images/HowToPublishDatasheetsInCkan3.png)
 
+You can also provide dynamic context information by providing a context broker
+URL and selecting the `fiware-ngsi` format. Once selected the `fiware-ngsi`
+format, three extra fields will appear allowing you to configure the tenant and
+the service path to use for accesing the context broker (you can leave them
+empty if the data is stored in the default tenant or service path). The last
+field allows you to indicate whether the context broker requires authentication
+(see the [Handling authorization and access control to APIs][]
```

Comment:
Yes, there is a tag at the end of the file

    [Handling authorization and access control to APIs]: /handling-authorization-and-access-control-to-apis/introduction/